src/cmstats.c

```diff
@@ -106,28 +112,32 @@ s_arithmetic_mean (const fty_proto_t *bmsg, fty_proto_t *stat_msg)
             fty_proto_type ((fty_proto_t*) bmsg),
             fty_proto_name ((fty_proto_t*) bmsg)
         );
-        return;
     }
```

Comment:
The change in behavior looks good, but instead of having a return variable that is always false except for one case, you can put 'return false' here and avoid indenting the rest of the function body.

Reply:
I changed it so that there's only one return statement, which is clearer IMHO than having a bunch of return statements sprinkled all over the place.

@geraldguillaume What's your opinion on it?

Reply:
I agree with @michal42